store/tikv/region_cache.go

```diff
@@ -254,7 +262,11 @@ func (c *RegionCache) GetStoreAddr(bo *Backoffer, id uint64) (string, error) {
 		return store.Addr, nil
 	}
 	c.storeMu.RUnlock()
+	return c.ReloadStoreAddr(bo, id)
+}
 
+// ReloadStoreAddr reload and save store's address
+func (c *RegionCache) ReloadStoreAddr(bo *Backoffer, id uint64) (string, error) {
```

Comment:
Need to use this method alone one day ?  Just like it may need reload store address forcibly in the feature. @disksing